src/shared/modules/dbMeta/dbMetaDuck.ts

```diff
@@ -128,6 +136,23 @@ export const getActiveDbName = (state: any) =>
  * Helpers
  */
 
+export const VERSION_FOR_EDITOR_HISTORY_SETTING = '4.3.0'
+
+export const versionHasEditorHistorySetting = (version: string) =>
+  semver.gte(version, VERSION_FOR_EDITOR_HISTORY_SETTING)
+
+export const supportsEditorHistorySetting = (state: any) =>
+  isEnterprise(state) && versionHasEditorHistorySetting(getVersion(state))
+
+export const shouldAllowOutgoingConnections = (state: any) =>
+  !isEnterprise(state) || getAllowOutgoingConnections(state)
+
+export const shouldRetainConnectionCredentials = (state: any) =>
+  !isEnterprise(state) || getRetainConnectionCredentials(state)
+
+export const shouldRetainEditorHistory = (state: any) =>
```

Comment:
it's a nit but could we move this to by be the other history getters? (so to line 146). Feel free to disregard this comment, just find it slightly more readable